scripts/inference-dev

```diff
@@ -50,7 +48,7 @@ fi
 
 eval "java" \
      "-DInferenceDevelLauncher.binary=${distDir} " \
-     "-DInferenceDevelLauncher.runtime.cp=${CFBuild}:${CFIBuild}:${OtherJars} " \
+     "-DInferenceDevelLauncher.runtime.cp=${CFBuild}:${CFIBuild} " \
```

Comment:
Without `OtherJars` the `SAT4jJars` won't be on the runtime classpath. Or are they discovered in some other way now?

Reply:
Actually, I didn't update this script correctly.  I'll fix it.